helm/higress/values.yaml

```diff
@@ -17,6 +17,8 @@ gateway:
     # If enabled, roles will be created to enable accessing certificates from Gateways. This is not needed
     # when using http://gateway-api.org/.
     enabled: true
+    # If enabled, gateway can use cross namespace secrets
+    crossNamespaceSecret: true
```

Comment:
Where is the parameter used for?

Reply:
The helm template will use this to determine whether to install the ClusterRole resource.

Reply:
@SpecialYang The option is removed to avoid ambiguity about the cross namespace usage of secret.